apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -54,6 +54,64 @@ export function initializeCircuitPlaygroundComponents(board) {
 }
 
 /**
+ * De-initializes any Johnny-Five components that might have been created
+ * by createCircuitPlaygroundComponents
+ * @param {Object} components - map of components, as originally returned by
+ *   createCircuitPlaygroundComponents.  This object will be mutated: Destroyed
+ *   components will be removed. Additional members of this object will be
+ *   ignored.
+ */
+export function destroyCircuitPlaygroundComponents(components) {
+  if (components.colorLeds) {
+    components.colorLeds.forEach(led => led.stop());
+  }
+  delete components.colorLeds;
```

Comment:
Actually, the reason I _don't_ want it in the existence check block is that we want to remove the key even if the value is falsy.

```javascript
var a = {};
var b = {foo: undefined};
Object.keys(a); // []
Object.keys(b); // ['foo']
```